src/database/seeds/relational/event/event-seed.service.ts

```diff
@@ -66,7 +66,7 @@ export class EventSeedService {
         startDate: eventData.startDate,
         endDate: eventData.endDate,
         maxAttendees: eventData.maxAttendees,
-        location: eventData.location ?? '',
```

Comment:
Location is still a 'string' with address name.

Reply:
Bring back location. What was the reason to remove it?

Reply:
This isn't addressed.  Stan needs the location